website/docs/zephyr-training/01_What_is_Zephyr/cmake.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 2
+description:
+    How Cmake is used in Zephyr
```

Comment:
nit. Should be `CMake` with a capital `M`, I believe? https://en.wikipedia.org/wiki/CMake

Reply:
Got it, thanks!